tools/cross_compile.go

```diff
@@ -0,0 +1,69 @@
+// +build none
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+)
+
+type Target struct {
+	os   string
+	arch string
+}
+
+var targets = []Target{
```

Comment:
There are two more tricks how this could be made even more concise, I think without losing readability in this case:
```
var targets = []struct{ os, arch string }{
  {"windows", "386"},
  {"windows", "amd64"},
  ...
}
```